### PR TITLE
Create test values for Decimal and Boolean types

### LINKED ISSUE #2859: Bug: Can't create CRUD version of service with Boolean or Decimal Prisma field types
After adding the new scenario fix https://github.com/redwoodjs/redwood/pull/2823 a bug was revealed—if your model data contains a Boolean or Decimal field type (as defined in prisma.schema), the code that turns the scenario data into a test throws an error. Turns out we weren't defining default values for Boolean or Decimal fields, so the new code attempts to call `replace` on `undefined`. 

The fix is to actually create default values for those fields before giving them to the template generator to turn into a test.

### PR DESCRIPTION
**Pull request by cannikin**
Closes #2859 #1969

### CODE REVIEW

File: packages/cli/src/commands/generate/sdl/sdl.js [hunk 1]
```diff
@@ -54,6 +54,7 @@ const modelFieldToSDL = (field, required = true, types = {}) => {
 
   const dictionary = {
     Json: 'JSON',
+    Decimal: 'Float',
```
Comment (dthyresson) [inline packages/cli/src/commands/generate/sdl/sdl.js:57]: Ah, there is a dictionary to map from Prisma to SDL scalar type. Nice.

**Comment by dthyresson** [comment]
LGTM!

**Comment by cannikin** [comment]
@thedavidprice No idea what the protocol is anymore! Do I go ahead and merge this? Do you or @jtoar merge? Cats and dogs, living together!

**Comment by jtoar** [comment]
@cannikin I think if you get an approval, just merge at will, barring anyone saying to hold for something. The idea being that the faster we merge something the faster we catch bugs before a release

**Comment by thedavidprice** [comment]
Ha! Nothing's changed with merging process. Fire away!

The only thing I need to know is if this is patch release worthy.

**Comment by jtoar** [comment]
@cannikin the project board isn't meant to block anything, just to give an overview of where we are--if we can expect a v1 RC soon, what's outstanding. Mostly just to answer questions and provide common ground for conversation!

**Comment by cannikin** [comment]
DONE! Whether it deserves a point release: generating SDL/services/scaffolds is totally broken if you have a Boolean or Decimal field in your schema. :( I think the tests themselves would have failed before, but at least they could have been generated. Now the generator itself throws an error (since 0.34.0).

**Pull request merged**